src/JWT/JWT.csproj

```diff
@@ -3,9 +3,9 @@
   <PropertyGroup>
     <TargetFrameworks>net462;netstandard1.3</TargetFrameworks>

```

Comment:
@abatishchev I believe you missed this line. The output still targets 4.6.2.

Reply:
Oh, right, redid. Please try a package from the most recent build (once it's done)